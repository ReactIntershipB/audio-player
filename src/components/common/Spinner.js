import React from 'react';
import { Icon } from 'antd';
import './Common.css';

const Spinner = () => {
    return (
        <div className="spinner-container"><Icon type="loading" theme="outlined" /></div>
    );
};

export default Spinner;
