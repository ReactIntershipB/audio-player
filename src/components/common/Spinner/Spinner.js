import React from 'react';
import { Spin } from 'antd';

export const Spinner = () => {
    return (
        <div className="spinner-container">
           <Spin size="large" />
        </div>
    );
};
