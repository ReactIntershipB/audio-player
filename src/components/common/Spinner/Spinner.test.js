import React from 'react';
import { Spinner } from './Spinner';
import renderer from 'react-test-renderer';

describe('Spinner', () => {
  it('to match snapshot', () => {
    const spinner = renderer
    .create(<Spinner />)
    .toJSON();

  expect(spinner).toMatchSnapshot();
  });
});
