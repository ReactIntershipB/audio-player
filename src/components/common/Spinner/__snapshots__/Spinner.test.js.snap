// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> to match snapshot 1`] = `
<div
  className="ant-spin ant-spin-lg ant-spin-spinning spinner"
>
  <span
    className="ant-spin-dot ant-spin-dot-spin"
  >
    <i />
    <i />
    <i />
    <i />
  </span>
</div>
`;
